Authorization/Authorization/Presentation/SSO/SSOHelper.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  SSOHelper.swift
+//  Authorization
+//
+//  Created by Rawan Matar on 02/06/2024.
+//
+
+import Foundation
+
+// https://developer.apple.com/documentation/ios-ipados-release-notes/foundation-release-notes
+
+/**
+    A Helper for some of the SSO preferences.
+    Keeps data under the UserDefaults.
+ */
+public class SSOHelper: NSObject {
+
+    public enum UserDefaultKeys: String, CaseIterable {
+        case cookiePayload
+        case cookieSignature
+        case userInfo
+        
+        var description: String {
+            switch self {
+            case .cookiePayload:
+                return "edx-jwt-cookie-header-payload"
+            case .cookieSignature:
+                return "edx-jwt-cookie-signature"
+            case .userInfo:
+                return "edx-user-info"
+            }
+        }
+    }
+
+    // MARK: - Singleton
+    
+    public static let shared = SSOHelper()
```

Comment:
please use Container in AppAssembly instead of using shared🙏

Reply:
sure 🫡